src/LogExpert.UI/Dialogs/LogTabWindow/LogTabWindowEventHandlers.cs

```diff
@@ -78,16 +82,16 @@ private void OnLogTabWindowClosing (object sender, CancelEventArgs e)
             _statusLineEventWakeupHandle.Set();
             _ledThread.Join();
 
-            IList<LogWindow.LogWindow> deleteLogWindowList = new List<LogWindow.LogWindow>();
+            IList<LogWindow.LogWindow> deleteLogWindowList = [];
             ConfigManager.Settings.AlwaysOnTop = TopMost && ConfigManager.Settings.Preferences.AllowOnlyOneInstance;
             SaveLastOpenFilesList();
 
-            foreach (LogWindow.LogWindow logWindow in _logWindowList)
+            foreach (var logWindow in _logWindowList)
             {
                 deleteLogWindowList.Add(logWindow);
             }
 
-            foreach (LogWindow.LogWindow logWindow in deleteLogWindowList)
+            foreach (var logWindow in deleteLogWindowList)
```

Comment:
This can be simplified and made more efficient by eliminating the intermediate list altogether, since we are not modifying _logWindowList during iteration.

The code can be changed to this:
```cs
foreach (var logWindow in _logWindowList.ToArray())
{
    RemoveAndDisposeLogWindow(logWindow, true);
}
```

Reply:
done